src/Core/src/Stream/IterableStream.php

```diff
@@ -8,8 +8,6 @@
  * Convert an iterator into a Stream.
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
- *
- * @internal
```

Comment:
Of all these stream classes, this IterableStream could stay internal if the S3 signer uses the factory instead. but other special streams can only be used directly.